specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tooling Service Client",
+    "description": "Client that can be used to access Tooling services to run linter, static and api validations.",
+    "version": "1.0"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "<TBD>",
+  "basePath": "/api/",
+  "schemes": [
+    "https", "http"
+  ],
+  "paths": {
+    "/tools/lint": {
+      "post": {
+        "tags": [
+          "Linter"
+        ],
+        "operationId": "Tools_Linter",
```

Comment:
Should this be more like "Linter_GetResults" or "Validation_Linter" or "Linter_ValidationResults"? Since this converts to the class and method in the SDK.

Reply:
Done